interface.go

```diff
@@ -90,10 +90,13 @@ type Job interface {
 	Lock(owner string, lockTimeout time.Duration) error
 	Unlock(owner string, lockTimeout time.Duration)
 
-	// Scope provides the ability to configure mutual exclusion for a job in a
-	// queue. When called, these methods do not actually take a lock; rather,
-	// they signal the intention to lock within the queue. It is invalid for end
-	// users to call SetScopes after the job has already dispatched.
+	// Scopes provide the ability to configure mutual exclusion for a job in a
```

Comment:
`provides`

Reply:
This is a little vague because it's talking about the concept of scopes in general. I clarified that the Scopes method returns the scopes.